test/unit/YawPitchControl/YawPitchControl.spec.js

```diff
@@ -2111,15 +2111,22 @@ describe("YawPitchControl", function() {
 		};
 
 		class MockRotationPanInput extends RotationPanInput {
-			constructor(el, options) {
-				super(el, options);
-
-				this._screenRotationAngle = {
-					getRadian: function() {
-						return glMatrix.toRadian(90); /* 90 degree */
-					},
-					unref: function() {
-						/* Do nothing */
+			setUseRotation(useRotation) {
```

Comment:
@happyhj how about mocking ScreenRotationAngle instead of overriding `setUseRotation`?